sdk/minio/src/helpers/client-options.schema.ts

```diff
@@ -1,12 +1,11 @@
-import { UrlSchema } from "@settlemint/sdk-utils/validation";
 import { z } from "zod";
 
 /**
- * Schema for validating client options for the Portal client.
+ * Schema for validating client options for the MinIO client.
  */
 export const ClientOptionsSchema = z.object({
   /** The URL of the MinIO instance to connect to */
-  instance: UrlSchema,
+  instance: z.string().url(),
```

Comment:
Why is this no longer a URL schema?

Reply:
@pmualaba had this error also, delete sdk/minio/node_modules directory (it happens because you have multiple versions of the zod module installed)

Reply:
should be fixed if you pull the latest changes from main and delete sdk/package/node_modules directory